cartography/intel/gcp/apigateway.py

```diff
@@ -698,26 +698,24 @@ def sync(
     load_apigateway_locations(neo4j_session, locations, project_id, gcp_update_tag)
     # Cleanup Locations
     cleanup_apigateway_locations(neo4j_session, common_job_parameters)
-    label.sync_labels(neo4j_session, locations, gcp_update_tag, common_job_parameters, 'apigateway_locations')
+    label.sync_labels(neo4j_session, locations, gcp_update_tag,
+                      common_job_parameters, 'apigateway_locations', 'GCPLocation')
     # API Gateway APIs
     apis = get_apis(apigateway, project_id, regions)
     load_apis(neo4j_session, apis, project_id, gcp_update_tag)
     for api in apis:
         load_apis_entity_relation(neo4j_session, api, gcp_update_tag)
     # Cleanup APIs
     cleanup_apis(neo4j_session, common_job_parameters)
-    label.sync_labels(neo4j_session, apis, gcp_update_tag, common_job_parameters, 'apis')
```

Comment:
we should call cleanup for apis as well.

Reply:
cleanup are called cleanup methods not deleted